server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -37,43 +37,64 @@
 import io.javalin.http.util.ContextUtil;
 import io.javalin.plugin.json.JavalinJson;
 
-
 /**
-* Tests the logic of the UserController
-*
-* @throws IOException
-*/
+ * Tests the logic of the UserController
+ *
+ * @throws IOException
+ */
+@SuppressWarnings({ "MagicNumber" })
 public class UserControllerSpec {
 
-  MockHttpServletRequest mockReq = new MockHttpServletRequest();
-  MockHttpServletResponse mockRes = new MockHttpServletResponse();
-
-  private UserController userController;
-
-  private ObjectId samsId;
-
-  static MongoClient mongoClient;
-  static MongoDatabase db;
-
-  static ObjectMapper jsonMapper = new ObjectMapper();
+  // The client and database that will be used
```

Comment:
I rearranged/restructred the fields and the setup/teardown code to try to get related pieces closer together. I don't know if that actually helped, but I found it easier to keep track of this way.